CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (simapp) [\#3874](https://github.com/cosmos/ibc-go/pull/3874) imp: refactor simapp to more closely resemble the style used in cosmos-sdk
```

Comment:
You can just remove this completely and git cliff will parse the squashed commit msg (PR title by default). Otherwise we end up with duplicate/similar changelog entries.